build/charts/theia/README.md

```diff
@@ -18,6 +18,12 @@ Kubernetes: `>= 1.16.0-0`
 
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
+| clickhouse.cluster.enable | bool | `false` | Determine whether to deploy ClickHouse as a cluster. |
+| clickhouse.cluster.installZookeeper | object | `{"size":"5Gi"}` | Specification to install a default ZooKeeper cluster. This is ignored if zookeeperHosts are provided. The default ZooKeeper cluster is deployed with 3 replicas. |
+| clickhouse.cluster.podDistribution | list | `[{"number":1,"topologyKey":"kubernetes.io/hostname","type":"MaxNumberPerNode"}]` | Affinity for the ClickHouse Pods. By default, it allows only one ClickHouse instance per Node. Please refer to <https://github.com/Altinity/clickhouse-operator/blob/master/docs/chi-examples/99-clickhouseinstallation-max.yaml> for other distributions. |
+| clickhouse.cluster.replicas | int | `2` | Number of ClickHouse replicas in each shard. |
+| clickhouse.cluster.shards | int | `2` | Number of ClickHouse shards in the cluster. |
```

Comment:
I think the default config creates 2 shards with 2 replicas each.
Can we default to a single shard with 2 replicas, as that would be enough for many small deployments, and use less resources in our CI?

Reply:
Got it. Update.